ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -453,6 +453,8 @@ literals:
 - zone_air_static_pressure_setpoint
 - zone_air_voc_concentration_sensor
 - zone_air_voc_concentration_setpoint
+- zone_air_occupied_cooling_temperature_setpoint
```

Comment:
these appear no longer used in this PR. please remove.

Reply:
Updated